lib/client/udp-tracker.js

```diff
@@ -63,8 +64,32 @@ UDPTracker.prototype._request = function (opts) {
   var self = this
   if (!opts) opts = {}
   var parsedUrl = url.parse(self.announceUrl)
+  if (!parsedUrl.port) {
+    parsedUrl.port = 80
+  }
+  var timeout
+  var proxySocket
```

Comment:
I'd add a comment here that explains the difference between socket and proxySocket
- If socks is enabled, what do they both do?
- If socks is disabled, is proxySocket null? etc